sqlglot/dialects/snowflake.py

```diff
@@ -137,6 +138,7 @@ class Tokenizer(Tokenizer):
             "TIMESTAMP_NTZ": TokenType.TIMESTAMP,
             "TIMESTAMP_TZ": TokenType.TIMESTAMPTZ,
             "TIMESTAMPNTZ": TokenType.TIMESTAMP,
+            "VARIANT": TokenType.SQL_VARIANT,
```

Comment:
can you refactor the data type to just be TokenType.VARIANT and fix TSQL?

currently the default is sql_variant, but the default should just be variant